src/curated-corpus/components/ScheduleHistory/ScheduleHistory.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { Grid, Typography, Collapse } from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { ApprovedCorpusItemScheduledSurfaceHistory } from '../../../api/generatedTypes';
+import { useStyles } from './ScheduleHistory.styles';
+import {
+  getCuratorNameFromLdap,
+  getScheduledSurfaceName,
+} from '../../helpers/helperFunctions';
+import { Button } from '../../../_shared/components';
+
+interface ScheduleHistory {
+  data: ApprovedCorpusItemScheduledSurfaceHistory[];
+}
+
+export const ScheduleHistory: React.FC<ScheduleHistory> = (
```

Comment:
It would be great to see comment blocks here for the interface prop and also for the component itself.